salt/modules/virt.py

```diff
@@ -145,6 +146,7 @@ def __esxi_auth():
                                     __esxi_auth(),
                                     0]],
         'qemu': [libvirt.open, [conn_str]],
+        'bhyve': [libvirt.open, ['bhyve:///system']],
```

Comment:
The whole connection mechanism has been rewritten recently. You don't need to add that anymore. See commit 2dfdc4eb4bdf9b168df53ee3416ebeb384d2247e for more details.